### PR TITLE
Adds section to test R file with dummy function

### PR DESCRIPTION
**Pull request by liz-fw**
Creates power function and a test plot. Created section for Erin's review to practice pull requests.

### CODE REVIEW

File: test-code.R [hunk 1]
```diff
@@ -12,4 +12,20 @@ square <- function(x) {
...
+plot(x, y, type = 'l')
+
+
+# for erin's review -------------------------------------------------------
+
+# practicing a pull request
+# adding a new function here to generalize the power function
+
+my_power_function <- function(values, exponent) {
+  # vinputs are a vector of values and an exponent
```
Comment (ErinCain) [inline test-code.R:24]: Please update vinputs to inputs.

File: test-code.R [hunk 2]
```diff
@@ -12,4 +12,20 @@ square <- function(x) {
...
+  output <- values^exponent
+  return(output)
+}
+
+# test new power function
+cubed <- my_power_function(x, 3)
+plot(x, cubed, type = 'l')
```
Comment (ErinCain) [inline test-code.R:31]: Try using ggplot to recreate the plot that you have here

File: test-code.R [hunk 3]
```diff
@@ -12,4 +12,20 @@ square <- function(x) {
 # example:
 x <- seq(1:10)
 y <- square(x)
-plot(x, y, type = 'l')
\ No newline at end of file
+plot(x, y, type = 'l')
+
+
+# for erin's review -------------------------------------------------------
```
Comment (ErinCain) [inline test-code.R:18]: Please see if you can add documentation to this function using Roxygen. This site should help: https://r-pkgs.org/man.html

Let me know if you have questions on how to do this.

**Review by ErinCain — COMMENTED** [review]
Hi Liz. Looking good. I added a few comments that I would like for you to address. Please use a separate commit to address each comment. Let me know if you want to work through the documentation stuff together. 

Thanks! (:

File: test-code.R (hunk 3 above)
Comment (ErinCain) [reply test-code.R:18]: To do this you will also need to restructure the test repository to be a package. I am happy to help walkthrough this process with you

Comment (liz-fw) [reply test-code.R:21]: Hi erin, I made the changes in a few separate commits so I think it should be good for final review!

Comment (ErinCain) [reply test-code.R:21]: Hi Liz. These commits look great. Thank you for keeping them small and giving informative descriptions. 

One more quick fix before I merge. The last commit I see that you added the r project file but I am not seeing the oxygen formatting, R folder, man folder, or DESCRIPTION files. Can you make sure that you push all of the files up that were on your computer? Let me know if you need help identifying why these didn't come up in the last commit.

Comment (liz-fw) [reply test-code.R:21]: Hi Erin, just pushed the rest of the files I had been missing!

Comment (ErinCain) [reply test-code.R:21]: Looks great. i will merge now!

**Pull request merged**